test/server/utils/runOpen.test.js

```diff
@@ -45,6 +45,48 @@ describe('runOpen util', () => {
       });
     });
 
+    it('on specify URL with page inside array', () => {
+      return runOpen(
+        'https://example.com',
+        { openPage: ['/index.html'] },
+        console
+      ).then(() => {
+        expect(opn.mock.calls[0]).toMatchInlineSnapshot(`
+          Array [
+            "https://example.com/index.html",
+            Object {
+              "wait": false,
+            },
+          ]
+        `);
+      });
+    });
+
+    it('on specify URL with multiple pages inside array', () => {
+      return runOpen(
+        'https://example.com',
+        { openPage: ['/index.html', '/index2.html'] },
+        console
+      ).then(() => {
+        expect(opn.mock.calls[0]).toMatchInlineSnapshot(`
+          Array [
+            "https://example.com/index.html",
+            Object {
+              "wait": false,
+            },
+          ]
+        `);
+        expect(opn.mock.calls[1]).toMatchInlineSnapshot(`
+          Array [
+            "https://example.com/index2.html",
+            Object {
+              "wait": false,
+            },
+          ]
+        `);
+      });
+    });
```

Comment:
don't use `toMatchInlineSnapshot`, use `toMatchSnapshot`, it is still unstable in some cases

Reply:
Done